examples/bme680/bme680_heating_profiles/bme680_heating_profiles.c

```diff
@@ -0,0 +1,126 @@
+/**
+ * Simple example with one sensor connected to I2C bus 0 and a sequence of
+ * heating profiles. The heating profile is changed with each cycle.
+ *
+ * Harware configuration:
+ *
+ *   I2C   +-------------------------+     +----------+
+ *         | ESP8266  Bus 0          |     | BME680   |
+ *         |          GPIO 5 (SCL)   ------> SCL      |
+ *         |          GPIO 4 (SDA)   ------- SDA      |
+ *         +-------------------------+     +----------+
+ */
+ 
+#include "espressif/esp_common.h"
+#include "esp/uart.h"
+
+#include "FreeRTOS.h"
+#include "task.h"
+
+// include communication interface driver
+#include "i2c/i2c.h"
+
+// include BME680 driver
+#include "bme680/bme680.h"
+
+// define I2C interface for BME680 sensors
+#define I2C_BUS         0
+#define I2C_SCL_PIN     GPIO_ID_PIN((5))
```

Comment:
There is no need to use GPIO_ID_PIN(...) macro here

Reply:
Thank you for your revire. Removed GPIO_ID_PIN(...) macro.